README.md

```diff
@@ -187,6 +189,15 @@ function onXRFrame(time, xrFrame) {
 // Layer interface
 //
 
+dictionary XRSubImage {
```

Comment:
I changed it to a dictionary since it's just a holder of information. 
If possible we should avoid adding classes to the global namespace

Reply:
I tend to agree with you, but I've heard from platform ergonomics folks/tag that the general pattern they expect from the web is dictionaries for inputs, classes for outputs. 🤷‍♀

Reply:
Looking at some specs (ie https://www.w3.org/TR/webrtc/), there are functions that return dictionaries.
It's a bit weird to introduce classes just to satisfy some platform convention. I'd really like to know why the tag wants this as it's not very javascript-y.

Reply:
I'll see if I can get some additional feedback on it.

Reply:
I can't find any part in WebRTC that does this? But yes, this is a platform convention.

Reply:
Furthermore, webidl isn't really _equipped_ to deal with this, I suspect browser codegen might even choke on this.

Remember that null dictionaries are equivalent to dictionaries with all empty fields, and we're returning a nullable in `getViewSubImage()`, and that's probably not going to work the way we want it to.

I think webidl parsers will choke on the `WebGLTexture?` below as well.

Reply:
> I can't find any part in WebRTC that does this? But yes, this is a platform convention.

See the getParameters call [here](https://www.w3.org/TR/webrtc/#rtcrtpsender-interface)

Reply:
> I think webidl parsers will choke on the `WebGLTexture?` below as well.

I keep getting confused about webidl. :-)

I just want to indicate that these are optional parameters. How do we do that? If we can't, we can just return them as null.

Reply:
Perhaps `optional WebGLTexture`. I'm not sure, like I said dictionaries aren't really designed to be returned, and the framework around them is optimized for the fields being supplied, not consumed.